INSTALL

```diff
@@ -36,6 +46,25 @@ pass its locations to the build script, e.g.::
   LDFLAGS=-L/usr/local/lib
   export CFLAGS LDFLAGS
 
+Create a Virtual Environment
+````````````````````````````
+Create the venv::
+
+    python3 -m venv gamera_venv
+
+Activate the venv::
+
+    source gamera_venv/bin/activate
+
+You should see something like: ``(gamera_venv) user@host:~$``
```

Comment:
This depends on the shell settings. Indirectly indicating that this is in the home directory does not look correct either.